cli/src/main/kotlin/utils/PackageConfigurationOption.kt

```diff
@@ -31,5 +31,12 @@ internal fun PackageConfigurationOption?.createProvider(): PackageConfigurationP
     when (this) {
         is PackageConfigurationOption.Dir -> SimplePackageConfigurationProvider.forDirectory(value)
         is PackageConfigurationOption.File -> SimplePackageConfigurationProvider.forFile(value)
-        null -> SimplePackageConfigurationProvider.EMPTY
+        null -> {
```

Comment:
As @tsteenbe already mentioned in another comment of this PR, how about defaulting to `package-configuration-dir` instead of to a file?

Reply:
Done.

Reply:
Thanks!